### DISCUSSION TITLE
Importer not reachable

### DISCUSSION THREAD

**Discussion by meinnamehier** [firefly-iii/firefly-iii#7664] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I installed Firefly and the importer via docker compose on an Unraid box.
Everything is running, but I can not reach the Importer via the designated port.
The relevant section of the docker-compose.yml:
  importer:
    image: fireflyiii/data-importer:latest
    hostname: importer
    networks:
      - firefly_iii
    restart: always
    ports:
      - '8181:8081'
    depends_on:
      - app
    env_file: .importer.env

But it is not reachable on port 8181; the logs show that the health check is running and it is successful.
The Firefly app is reachable.
FIREFLY_III_URL=http://app:8080
VANITY_URL is left empty
I am not quite sure what I am doing wrong.

### Debug information

Debug information generated at 2023-06-17 12:27:38 Europe/Berlin for Firefly III version **6.0.13**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.13 |
| Firefly III API | 2.0.4 |
| Build | 506, 11-06-2023 16:24:57 UTC |
| Base Build | 6, 10-06-2023 16:13:33 CEST |
| DB version | 19 (exp. 19) |
| PHP | `8.2.7` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Berlin |
| Browser TZ | Europe/Berlin |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 67108864 (64 MB) |
| Interface | apache2handler |
| Default language | de_DE |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start |  |
| Session end |  |
| Session first |  |
| User ID | 1 |
| User language | de_DE |
| User locale | de_DE |
| Attempt at "de_DE.utf8" | 'de_DE.utf8' |
| Attempt at "de_DE.UTF-8" | 'de_DE.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/113.0 |

### Expected behaviour

I can reach the importer on port 8181 on the server

### Steps to reproduce

1. Setup everything according to the docks with docker compose
2. Change external port of app to 8180 and importer to 8181

**Comment by meinnamehier** [comment]
Okay, i figured it out. Of course, I had the port assignments wrong,
it works with 
   ports:
      - '8181:8080'
Silly me.

**Reply by JC5** [reply]
Classic. Glad you found it!
